test/commands/create.test.ts

```diff
@@ -0,0 +1,452 @@
+      profileName: 'profileFromArgs',
+      lastName: 'User',
+      timeZoneSidKey: 'America/Los_Angeles',
+    };
+    // @ts-ignore
+    $$.SANDBOX.stub(DefaultUserFields, 'create').callsFake(() => {
+      const defaultFields = new DefaultUserFields({ templateUser: userInfo.username });
+      // @ts-ignore
+      defaultFields.userFields = userInfo;
+      return Promise.resolve(defaultFields);
+    });
+    $$.SANDBOX.stub(AuthInfo.prototype, 'getFields').returns({
+      userId: newUserId,
+      username: '1605130295132_test-j6asqt5qoprs@example.com',
+      alias: 'testAlias',
+    });
+    $$.SANDBOX.stub(Org.prototype, 'getOrgId').returns(testOrg.orgId);
+
+    if (throws.license) {
+      $$.SANDBOX.stub(User.prototype, 'createUser').throws(new Error('LICENSE_LIMIT_EXCEEDED'));
+      $$.SANDBOXES.CONNECTION.stub(Connection.prototype, 'query').resolves({
+        records: [{ Name: 'testName' }],
+        done: true,
+        totalSize: 1,
+      });
+    } else if (throws.duplicate) {
+      $$.SANDBOX.stub(User.prototype, 'createUser').throws(new Error('DUPLICATE_USERNAME'));
+    } else {
+      $$.SANDBOX.stub(User.prototype, 'createUser').callsFake(async (): Promise<AuthInfo> => {
+        const authInfo = await AuthInfo.create({ username: newUser.username });
+        newUser.orgId = (await Org.create({ aliasOrUsername: testOrg.username })).getOrgId();
+        return authInfo.save(newUser);
+      });
+    }
+
+    if (readsFile) {
+      $$.SANDBOXES.CONNECTION.stub(Connection.prototype, 'query').resolves({
+        records: [{ Id: '12345678' }],
+        done: true,
+        totalSize: 1,
+      });
+      $$.SANDBOX.stub(Logger.prototype, 'debug');
+    }
+  }
+
+  it('will handle a merge multiple permsets and profilenames from args and file (permsets from args)', async () => {
+    await prepareStubs({}, { profilename: 'profileFromFile', permsets: ['perm1', 'perm2'] });
+    $$.SANDBOX.stub(User.prototype, 'assignPassword').resolves();
+    const expected = {
+      orgId: testOrg.orgId,
+      permissionSetAssignments: ['permCLI', 'permCLI2'],
+      fields: {
+        alias: 'testAlias',
+        email: 'defaultusername@test.com',
+        emailencodingkey: 'UTF-8',
+        id: newUserId,
+        languagelocalekey: 'en_US',
+        lastname: 'User',
+        localesidkey: 'en_US',
+        generatepassword: true,
+        profileid: '12345678',
+        profilename: 'profileFromArgs',
+        timezonesidkey: 'America/Los_Angeles',
+        username: '1605130295132_test-j6asqt5qoprs@example.com',
+      },
+    };
+    const createCommand = new UserCreateCommand(
+      [
+        '--json',
+        '--target-org',
+        testOrg.username,
+        '--target-dev-hub',
+        'devhub@test.com',
+        "permsets='permCLI, permCLI2'",
+        'generatepassword=true',
+      ],
+      {} as Config
+    );
+    const result = await createCommand.run();
+    expect(result).to.deep.equal(expected);
+  });
+
+  // test
```

Comment:
Any need for these? Should reenable or remove?

Reply:
Will reenable - thanks